Library/.rubocop.yml

```diff
@@ -8,7 +8,15 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+
 Homebrew/CorrectBottleBlock:
+  Include:
+    - '**/Taps/homebrew/**/*.rb'
```

Comment:
> If desc can be added in the above files, then we don't have to repeat Include in .rubocop.yml

Why not just check if the file path contains `/Library/Homebrew/`, and exclude those? Didn't we have this approach already before?